scripts/airdrop.remove-principals.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Get list of principals to remove from the airdrop
+
+set -eExuo pipefail
```

Comment:
remove

Reply:
For script that change the state and are not called during deployment of the canister I like to see the command that is effectively being run!

Reply:
Oki doki

Reply:
I kept only `set -x` to see the command and removed the rest!